scripts/mann_kendall_cli.py

```
@@ -0,0 +1,82 @@
+
+"""
+Command line interface for Mann Kendall Automated.
+Allows processing Excel files via command line and saving results.
+"""
+
+import argparse
+import os
+import sys
+from pathlib import Path
+
+import pandas as pd
```

Ruff (0.8.2)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)